### PR TITLE
Adding unit test for Version method

### PR DESCRIPTION
**Pull request by Pensu**
Fixes https://github.com/openebs/openebs/issues/1922

### CODE REVIEW

File: pkg/install/v1alpha1/version_test.go [hunk 1]
```diff
@@ -40,3 +41,29 @@ func TestIsCurrentVersionValid(t *testing.T) {
...
+                        expversion: "invalid.version",
+                },
+        }
+
+        for name, tt := range tests {
+                t.Run(name, func(t *testing.T) {
+                        validatedVersion := Version(string(tt.version))
+                        if !reflect.DeepEqual(string(validatedVersion), tt.expversion) {
+                                t.Errorf("Version error %v", validatedVersion)
+                        }
```
Comment (prateekpandey14) [inline pkg/install/v1alpha1/version_test.go]: @Pensu thanks for the PR ..... can we make error more clearer. for example:
t.Errorf ( "version error, got version: %v , expected version: %v ", validatedVersion, tt.expversion)

Comment (Pensu) [reply pkg/install/v1alpha1/version_test.go]: @prateekpandey14 Made the changes. PTAL.

File: pkg/install/v1alpha1/version_test.go [hunk 2]
```diff
@@ -40,3 +41,29 @@ func TestIsCurrentVersionValid(t *testing.T) {
 		})
 	}
 }
+
+func TestVersion(t *testing.T) {
+        tests := map[string]struct {
+                version    string
+                expversion string
```
Comment (princerachit) [inline pkg/install/v1alpha1/version_test.go]: `expVersion` or `expectedVersion`

File: pkg/install/v1alpha1/version_test.go [hunk 3]
```diff
@@ -40,3 +41,29 @@ func TestIsCurrentVersionValid(t *testing.T) {
 		})
 	}
 }
+
+func TestVersion(t *testing.T) {
+        tests := map[string]struct {
+                version    string
```
Comment (princerachit) [inline pkg/install/v1alpha1/version_test.go]: this is also good but maybe `inputVersion` is better. what do you think @prateekpandey14 ?

File: pkg/install/v1alpha1/version_test.go (hunk 2 above)
Comment (Pensu) [reply pkg/install/v1alpha1/version_test.go]: @princerachit Just wanted to keep it short, so abbreviated expected to exp, you want me to change it?

File: pkg/install/v1alpha1/version_test.go [hunk 4]
```diff
@@ -40,3 +41,29 @@ func TestIsCurrentVersionValid(t *testing.T) {
...
+                "case 2": {
+                        version:    "",
+                        expversion: "invalid.version",
+                },
+        }
+
+        for name, tt := range tests {
+                t.Run(name, func(t *testing.T) {
+                        validatedVersion := Version(string(tt.version))
+                        if !reflect.DeepEqual(string(validatedVersion), tt.expversion) {
```
Comment (AmitKumarDas) [inline pkg/install/v1alpha1/version_test.go]: Please avoid reflect to do a string comparison.

**Pull request merged**